third_party/docfx/templates/devsite/common.js

```diff
@@ -62,6 +64,42 @@ function isRelativePath(path) {
     return !exports.isAbsolutePath(path);
 }
 
+var titlePrefixForTypeProperty = {
+    "package": "Package",
+    "subpackage": "Package",
+    "namespace": "Namespace",
+    "module": "Module",
+    "class": "Class",
+    "struct": "Struct",
+    "interface": "Interface",
+    "enum": "Enum",
+    "delegate": "Delegate",
+    "static field": "Static Field",
+    "static method": "Static Method",
+    "constructor": "Constructor",
+    "field": "Field",
+    "property": "Property",
+    "method": "Method",
+    "event": "Event",
+    "operator": "Operator",
+    "eii": "Explict Interface Implementation",
+    "variable": "Variable",
+    "typealias": "Type Alias",
+    "annotationtype": "Annotation Type",
+    "exception": "Exception"
+}
+
+function getTitleForTypeProperty(type, name, uid) {

```

Comment:
Will you add a comment saying why we have this function?

Reply:
Done.